File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-define-and-select-ranges-in-documents_7.cs
            if (this.Sentences.Count >= 2) 
            {
                object startLocation = this.Sentences[2].Start; 
                object endLocation = this.Sentences[2].End; 

                // Supply a Start and End value for the Range. 
                rng = this.Range(ref startLocation, ref endLocation); 

                // Select the Range.
                rng.Select();
            }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-define-and-select-ranges-in-documents_7.vb
        If Me.Sentences.Count >= 2 Then

            Dim startLocation As Object = Me.Sentences(2).Start
            Dim endLocation As Object = Me.Sentences(2).End

            ' Supply a Start and End value for the Range.
            rng = Me.Range(Start:=startLocation, End:=endLocation)

            ' Select the Range
            rng.Select()
        End If